examples/quad/Cargo.toml

```diff
@@ -0,0 +1,72 @@
+[package]
```

Comment:
Why do we need the quad example under a different cargo crate?

Reply:
I cannot find a configuration to use the same for native and web.
Nothing was explained to run web examples.
If you find something I'm very interested.

Reply:
You are right, the README of `/examples` needs to explain how to build for the web. @grovesNL didn't we have that somewhere?

Reply:
I'm on my phone so I can't get a link to the header, but it's under the examples README

Reply:
But for me the `examples/Cargo.toml` cannot provide the right target.
In native mode it generate a binary.
For Web, the crate requires to be a library...

Reply:
It's possible to build binary targets or examples with wasm-bindgen using the instructions there - we don't use wasm-pack or stdweb

Reply:
I found nothing to run the code using `wasm-bindgen` in development mode.
Nothing mentioned here: https://github.com/gfx-rs/gfx#example

And [paint example](https://github.com/rustwasm/wasm-bindgen/tree/master/examples/paint) that use le `#[wasm_bindgen(start)]` requires to add:
- In the `Cargo.toml`
```rust
[lib]
crate-type = ["cdylib"]
```
- start webpack (using `npm run serve`)

I have't found something to run `wasm-bindgen` without WebPack ...
That's why for the native, the `bin` works. But for Web example it's not enough.

Reply:
The instructions are located at https://github.com/gfx-rs/gfx/blob/master/examples/README.md#running-quad-with-webglwebassembly

Reply:
Thank you for the link.
I think it's not a real strait forward example setup...
Lot of operations are required to simply open the render.
And for development purpose it's quasi impossible to run and copy file every times ...

A minima I think it can be awesome to add a mention in the [root README](https://github.com/gfx-rs/gfx#example) to redirect to the URL.

I have opened an [issue](https://github.com/rustwasm/wasm-bindgen/issues/2359) on `wasm-bindgen` projet.

Reply:
Ok so I will revert example changes.